pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRSQL.scala

```diff
@@ -0,0 +1,12 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{SQLContext}
+import org.apache.spark.sql.types.{StructType}
+
+object sparkRSQL {
```

Comment:
I think `sparkRSQL` (as well as the file name) is confusing. Can we simplify to not risk being interpreted as a new concept? `object sql` might do.